openedx/adg/admin_dashboard/admin_settings/tests/test_views.py

```diff
@@ -0,0 +1,55 @@
+"""
+Tests for `admin_settings` app views
+"""
+import pytest
+
+from common.djangoapps.student.tests.factories import GroupFactory, UserFactory
+from openedx.adg.admin_dashboard.admin_settings.models import Invitation
+from openedx.adg.admin_dashboard.admin_settings.tests.factories import InvitationFactory
+from openedx.adg.admin_dashboard.admin_settings.views import AdminAccountListView, PendingInvitationListView
+from openedx.adg.lms.applications.constants import ADG_ADMIN_GROUP_NAME
+from openedx.adg.lms.applications.tests.factories import BusinessLineFactory
+from openedx.core.djangolib.testing.utils import skip_unless_lms
+
+
+@pytest.mark.parametrize('total_admins, expected_count', [
+    (5, 10),
+    (10, 20),
```

Comment:
I don't understand. How is the expected count double the total admins? For each admin account, we are creating a an active user and an inactive user. The inactive user will not be fetched in the queryset, so shouldn't the total admins and expected count be the same?

Reply:
Actually I am creating active ADG Admin and BU Admin each time in loop. In-active as well So 1 ADG admin and 1 BU admin which means 2 for each iteration.